dist/angular-localization.js

```diff
@@ -21,8 +21,7 @@ angular.module('ngLocalize.InstalledLanguages', [])
     });
 angular.module('ngLocalize')
     .service('locale', ['$injector', '$http', '$q', '$log', '$rootScope', '$window', 'localeConf', 'localeEvents', 'localeSupported', 'localeFallbacks', function ($injector, $http, $q, $log, $rootScope, $window, localeConf, localeEvents, localeSupported, localeFallbacks) {
-        var TOKEN_REGEX = new RegExp('^[\\w\\.-]+\\.[\\w\\s\\.-]+\\w(:.*)?$'),
-
+        var TOKEN_REGEX = (localeConf.validTokens | new RegExp('^[\\w\\.-]+\\.[\\w\\s\\.-]+\\w(:.*)?$')),
```

Comment:
Hmm, something went wrong here. I finally made a || check but that change was mistakenly not comitted. My mistake! But I think we need a check to not force users, who are overriding the parameters already, from having to specify the validTokens parameter when they upgrade to the latest version?